es6-shim.js

```diff
@@ -1438,7 +1438,7 @@
       if (number === 0) {
         return 32;
       }
-      return 32 - (number).toString(2).length;
+      return 31 - Math.floor(Math.log(number) * Math.LOG2E + (1 / 2 / 4294967296));
```

Comment:
That's why I'm asking - if it's hard for you, it'll be infinitely harder for me :-) and the name will help with future understanding.